src/Products/urban/migration/update_270.py

```diff
@@ -461,3 +463,59 @@ def hide_habitation_tab_in_licence_config(context):
                 licence_config.setTabsConfig(updated_config)
 
     logger.info("upgrade step done!")
+
+
+def add_work_related_index(context):
+    logger = logging.getLogger("urban: Add work related index")
+    logger.info("starting upgrade steps")
+
+    setup_tool = api.portal.get_tool("portal_setup")
+    setup_tool.runImportStepFromProfile("profile-Products.urban:urbantypes", "catalog")
+
+    catalog = api.portal.get_tool("portal_catalog")
+    licence_brains = catalog(object_provides=IGenericLicence.__identifier__)
+    for count, brain in enumerate(licence_brains):
```

Comment:
To avoid too much computation, can you please change this upgrade to simply reindex the two new indexes ?